### DISCUSSION TITLE
Resorting sequence of images based on text or sentence

### DISCUSSION THREAD

**Discussion by Tortoise17** [ImageMagick/ImageMagick#7189] [Help]
### ImageMagick version

7

### Operating system, version and so on

Linux

### Description

I have some images as frames in a folder which corresponds to a shots / video. I have text which is related to the frames, but not organized. Is there a way to rearrange sequence of images or  shots based on the sentence .. like sentence matching or alignment based on the frames sequence.

**Comment by snibgo** [comment]
You could rename the files to be whatever text you want to sort on. Or create symbolic links, using the text.

**Comment by Tortoise17** [comment]
Yes, this is manual way. Is there any automated way to do so?

**Reply by fmw42** [reply]
Imagemagick does not do text processing. You will need to script something.
